doc/README.md

```diff
@@ -212,3 +212,58 @@ To verify if the device supports Tap to Pay on iPhone:
 ```
 
 > **Note:** These methods are only available on iOS. Calling them on Android will result in an error.
+
+---
+
+## 📡 Offline Payments (Beta)
+
+The Mobile Payments SDK for Flutter supports taking payments offline. This feature is currently in **Beta** and requires Square sellers to opt in.
```

Comment:
Nit: this makes it sound like "opt in to beta," but in fact sellers need to opt in anyway (it's a liability issue; if an offline payment is later declined, Square doesn't pay for the failed payment, that's on the seller).  I would probably separate the two:

> This is currently a **Beta** feature in the SDK.  Additionally, use of offline payments requires Square sellers to opt in and agree to the terms of the feature.